packages/fxa-settings/src/pages/Signin/index.tsx

```diff
@@ -0,0 +1,244 @@
+                    Continue to{' '}
+                    <span>
+                      <ServiceLogo
+                        className="inline"
+                        aria-label={localizedLogoAriaLabel}
+                      />
+                    </span>
+                  </span>
+                </FtlMsg>
+              )}
+              {!isPocketClient && !ServiceLogo && (
+                <FtlMsg
+                  id="signin-subheader-without-logo"
+                  vars={{ serviceName }}
+                >
+                  <span className="card-subheader">
+                    Continue to {serviceName}
+                  </span>
+                </FtlMsg>
+              )}
+            </>
+          </>
+        )}
+      </div>
+      <section>
+        {/* Alerts and success messages originally went here */}
+        <div className="mt-9">
+          {/* When we get to the functionality stage, we can probably replace this with the Avatar component in Settings*/}
+          <div className="mx-auto h-24 w-24 mobileLandscape:h-40 mobileLandscape:w-40"></div>
+          <div className="my-5 text-base break-all">{email}</div>
+        </div>
+        <form noValidate {...{ onSubmit }}>
+          <input type="email" className="email hidden" value={email} disabled />
+
+          {isPasswordNeeded && (
+            <InputPassword
+              anchorStart
+              className="mb-5 text-start"
+              label={localizedPasswordFormLabel}
+              hasErrors={error.length > 0}
+              errorText={error}
+              tooltipPosition="bottom"
+              required
+              autoFocus
+              onChange={(e) => setPassword(e.currentTarget.value)}
+            />
+          )}
+          {/* This non-fulfilled input tricks the browser, when trying to
+              sign in with the wrong password, into not showing the doorhanger.
+           */}
+          <input className="hidden" required />
+
+          <div className="flex">
+            <FtlMsg id="signin-button">
+              <button className="cta-primary cta-xl" type="submit">
+                Sign in
+              </button>
+            </FtlMsg>
+          </div>
+        </form>
+        <div className="text-grey-500 my-5 text-xs">
+          {isPocketClient ? (
+            <FtlMsg id="firefox-and-pocket-tos">
+              <p className="text-xs">
+                By proceeding, you agree to:
+                <br />
+                Pocket’s{' '}
+                <LinkExternal
+                  className="link-grey"
+                  href="https://getpocket.com/tos/"
+                >
+                  Terms of Service
+                </LinkExternal>{' '}
+                and{' '}
+                <LinkExternal
+                  className="link-grey"
+                  href="https://getpocket.com/privacy/"
+                >
+                  Privacy Notice
+                </LinkExternal>
+                <br />
```

Comment:
I was going to say that these should probably be wrapped in a `span` set to `display: block` instead of using breaks since screenreaders will read all of these as separate elements, but with the colon, I can see how that can make sense. However, that kind of makes me think this should be an unordered list instead - picture this with like 5 different things someone needs to "agree" to. I'd say this can be done in a follow up but it seems directly tied to the FTL, so hmm.

Going to ping @sardesam for a sanity check here.

Reply:
Meghan and I pinged a little this morning, and we've concluded that `ul` is going to be our best option here. It gives the best of both worlds: there's separation between each line for screenreaders, and also groups the section together, so if screenreader users come across the title of the `ul` and don't care to read its content, they can skip it.

It would be really nice if each piece of this could have its own l10n ID and string, but I'm not sure if that's possible with the start of the sentence beginning with "you agree to:`.